prover/vk_setup_data_generator_server_fri/src/vk_commitment_helper.rs

```diff
@@ -17,11 +18,10 @@ pub fn write_contract_toml(contract_doc: Document) -> anyhow::Result<()> {
 pub fn read_contract_toml() -> anyhow::Result<Document> {
     let path = get_contract_toml_path();
     let toml_data = std::fs::read_to_string(path.clone())
-        .with_context(|| format!("contract.toml file does not exist on path {path}"))?;
+        .with_context(|| format!("contract.toml file does not exist on path {path:?}"))?;
     toml_data.parse::<Document>().context("invalid config file")
 }
 
-pub fn get_contract_toml_path() -> String {
-    let zksync_home = std::env::var("ZKSYNC_HOME").unwrap_or_else(|_| "/".into());
-    format!("{}/etc/env/base/contracts.toml", zksync_home)
+pub fn get_contract_toml_path() -> PathBuf {
+    workspace_dir_or_current_dir().join("../etc/env/base/contracts.toml")
```

Comment:
double-checking: does `join` work well with `..`?

Reply:
Yes, it does, it's covered by tests in the crate.